pkgs/applications/misc/snagboot/default.nix

```diff
@@ -0,0 +1,75 @@
+{ lib
+, stdenv
+, pythonRelaxDepsHook
+, python3
+, snagboot
+, testers
+, gitUpdater
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "snagboot";
+  version = "1.0";
+  format = "pyproject";
+
+  src = python3.pkgs.fetchPypi {
```

Comment:
I believe it is being deprecated in favor of the toplevel one. But keep it for now.

Reply:
I used this because we mix python3 and non-python pkgs below.

Reply:
Fixed.